docs/contributing/docs.md

```diff
@@ -2,186 +2,504 @@
+### Reporting violations
+To report violations contact the Crossplane maintainers at `info@crossplane.io`
+or the CNCF at `conduct@cncf.io`.
 
+## Docs source 
+Crossplane documentation lives in two repositories:
 
-### Directory structure
-The relevant directories of the [docs repository](https://github.com/crossplane/crossplane.github.io) are:
-| Directory | Purpose |
-| ----- | ---- |
-| `content` | Markdown files for all individual pages. |
-| `static/images` | Location for all image files.  |
-| `themes` | HTML templates and Hugo tooling. |
+* Crossplane documentation source is in the [Crossplane
+repository](https://github.com/crossplane/crossplane) `/docs` directory. 
 
-All markdown content exists within the `/content` directory.   
-All image files exist within `static/images`.
+* The Crossplane docs website is in the
+  [crossplane.github.io](https://github.com/crossplane/crossplane.github.io)
+repository.
 
-### Run Hugo
-* Follow the [Hugo documentation](https://gohugo.io/getting-started/installing/) to install Hugo.
-* Run `hugo server`
-* Go to [http://localhost:1313](http://localhost:1313)
+Use `crossplane/crossplane` for documentation contributions.  
+Use `crossplane/crossplane.github.io` for local development or updates involving
+HTML, CSS or Hugo.
 
-Crossplane documentation uses [Hugo v0.101.0](https://github.com/gohugoio/hugo/releases/tag/v0.101.0), but newer versions are compatible.
+### Licensing
+The Crossplane documentation is under the [Creative Commons
+Attribution](https://creativecommons.org/licenses/by/4.0/) license. CC-BY allows
+reuse, remixing and republishing of Crossplane documentation with attribution to
+the Crossplane organization.
 
-### Documentation issues and feature requests
-Use [GitHub Issues](https://github.com/crossplane/crossplane.github.io/issues) to report a problem or request documentation content.
+### Issues and feature requests
+Open an [issue](https://github.com/crossplane/crossplane/issues)
+to report a problem or request documentation content.
 
 ## Contributing
 
-The `/content` directory contains all documentation pages.  
-Each version of Crossplane software maintains a unique directory within `content`.
+Documentation content contributions are always welcome.
 
-### Front matter
-Each page contains metadata called [front matter](https://gohugo.io/content-management/front-matter/). Each page requires front matter to render.
+The Crossplane documentation source is in the [Crossplane
+repository](https://github.com/crossplane/crossplane) `/docs` directory. 
 
-```yaml
----
-title: "Troubleshooting Crossplane"
-weight: 610
----
-```
+Crossplane recommends using the [website
+repository](https://github.com/crossplane/crossplane.github.io) for local development.
+When a contribution is ready, submit a pull request to the [Crossplane
+repository](https://github.com/crossplane/crossplane).
```

Comment:
Why do we recommend this? You can run the docs site from this repo as outlined in https://github.com/crossplane/crossplane/issues/3461#issuecomment-1324536727. I believe the justification there was that folks may not have `make` installed, but it is going to be installed by default in many distributions. That means they only have to run one command, which feels much simpler than jumping between multiple repositories.

Reply:
>default in many distributions.

But not on mac or Windows.

Reply:
They are eventually having to come and interact with this repo, where we ask folks to run `make` commands prior to submitting a PR. I am all for reducing any barrier to entry, but I don't think that this is actually accomplishing that goal as it requires more steps rather than less.

Reply:
Suggestion removed.

Reply:
Thanks. I feel like the current status is still pointing folks towards using the Crossplane website repo, and is not suggesting using any of the machinery that is put in place there. We have single commands that ensure you have the correct version of hugo installed, etc. and allow for very quick iteration. Is the reason for not recommending any of this in order to avoid using `make`? If I was helping someone contribute I would say:
1. Clone the Crossplane repo
2. Make changes
3. Run `make docs.run`

This will ensure you have the latest version of the docs site and are using the correct version of hugo. Is there anything we could change about this machinery to make you more inclined to suggest it?

Reply:
You're a beat ahead of me. I was just walking through this workflow to add instructions but ran into https://github.com/crossplane/crossplane/issues/3487.

If this is resolved I'll put in detailed steps on the make process.

(also our make configuration [doesn't support Windows](https://github.com/upbound/build/blob/master/makelib/common.mk#L99))

Reply:
@plumbis opened https://github.com/crossplane/crossplane/pull/3488 to fix -- you can consider be signed up to fix any issues with improving the docs workflow in this repo since I am pushing so hard for using it :)

Reply:
Development section rewritten to cover make and local builds. https://github.com/crossplane/crossplane/pull/3486/files#diff-93426d953f03b733adbbb574ffd2717d0892773a464c0d301d1b6c9095d67b15R58